device/celestica/x86_64-cel_e1031-r0/sonic_platform/chassis.py

```diff
@@ -80,39 +65,41 @@ def get_base_mac(self):
         except KeyError:
             raise KeyError("Base MAC not found")
 
-    def get_component_versions(self):
+    def get_component_list(self):
```

Comment:
It seems this might be useful for other vendors. Would it be better to move this function and the `component_list` variable to the base class?

Reply:
Yes, I think so